app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss

```diff
@@ -93,6 +93,13 @@
   font-size: px(40);
 }
 
+@media (max-width: 425px){
```

Comment:
Any tips on how to rework the 'magic' would be appreciated.

'white-space: normal' is so that the submit button text wraps when the screen isn't wide enough.
The text on the button is 'Find out how much you can borrow'.